### PR TITLE
Added missing step to custom build documentation, clarified additional details.

### PR DESCRIPTION
**Pull request by coreygo**
Additional details here: https://github.com/processing/p5.js/issues/3425#issuecomment-450760491

### CODE REVIEW

**Comment by lmccart** [comment]
thank you!

File: developer_docs/custom_p5_build.md [hunk 1]
```diff
@@ -14,13 +14,14 @@ Currently, the usage is through invoking a Grunt task manually from the command
 git clone https://github.com/processing/p5.js.git
 cd p5.js
 npm install
-grunt combineModules:module_x:module_y
+npm run grunt
+npm run grunt combineModules:module_x:module_y
 ```
 
-Here, `module_n` refers to the name of the module which you want to select. Multiple modules must be passed as shown above. Also, these modules must have the same name is their folders in `/src` directory to work correctly.
+Here, `module_n` refers to the name of the module which you want to select. Multiple modules must be passed as shown above. Also, these modules must have the same name is their folders in `/src` directory to work correctly. While `core` is included by default, `core/shape` needs to be included for shapes like line() and other core features to work.
```
Comment (coreygo) [inline developer_docs/custom_p5_build.md]: Erroops, just noticed I missed updating "…name **is** their folders…", it should be "…name **as** their folders…" 🔤 ✅

**Comment by lmccart** [comment]
@coreygo thanks I've made the change

**Comment by coreygo** [comment]
Cheers, have a good one! 😄

**Pull request merged**